tools/shared.py

```diff
@@ -2299,6 +2299,12 @@ def reconfigure_cache():
   global Cache
   Cache = cache.Cache(debug=DEBUG_CACHE)
 
+# Temporary workaround for SINGLE_FILE, pending refactoring
+class FilenameReplacementStrings:
+  WASM_TEXT_FILE = 'FILENAME_REPLACEMENT_STRINGS_WASM_TEXT_FILE'
```

Comment:
let's add `{{{ ` and ` }}}`, as that's the convention we use for such replacement strings elsewhere